packages/website/src/components/Playground.module.css

```
@@ -37,3 +42,67 @@
   top: var(--ifm-navbar-height);
   z-index: calc(var(--ifm-z-index-fixed) - 1);
 }
+
+.tabContainer {
+  display: flex;
+  justify-content: space-between;
+  background: var(--playground-main-color);
+  border-bottom: 1px solid var(--playground-secondary-color);
+}
+
+.tabCode {
+  height: calc(100%);
+}
+
+.tabStyle {
+  border: none;
+  border-right: 1px solid var(--playground-secondary-color);
+  background: var(--playground-main-color);
+  color: var(--ifm-color-emphasis-600);
```

#8D949E doesn't have enough color contrast to be accessible against a white background. #606770 does, though. https://webaim.org/resources/contrastchecker/ 

```suggestion
  color: var(--ifm-color-emphasis-700);
```